vpr/src/route/partition_tree.h

```diff
@@ -83,9 +86,14 @@ class PartitionTree {
     /** Access root. Shouldn't cause a segfault, because PartitionTree constructor always makes a _root */
     inline PartitionTreeNode& root(void) { return *_root; }
 
+    void update_nets(const std::vector<ParentNetId>& nets);
+  
+    void clear_vnets(void);
+
   private:
     std::unique_ptr<PartitionTreeNode> _root;
-    std::unique_ptr<PartitionTreeNode> build_helper(const Netlist<>& netlist, const std::vector<ParentNetId>& nets, int x1, int y1, int x2, int y2);
+    std::unordered_map<ParentNetId, PartitionTreeNode*> _net_to_ptree_node;
+    std::unique_ptr<PartitionTreeNode> build_helper(const Netlist<>& netlist, const std::unordered_set<ParentNetId>& nets, int x1, int y1, int x2, int y2);
```

Comment:
Doxygen comment. Explain all parameters (nets are those to consider adding to this node, and x1,y1,x2,y2 are the bounding box of the partition node).